sdks/java/testing/test-utils/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
   implementation library.java.http_client
   implementation library.java.http_core
   implementation library.java.slf4j_api
+  provided library.java.jmh_core
```

Comment:
I don't understand the motivation to use `provided` over `implementation`?

Reply:
Motivation was to not include `jmh-core` on every classpath that depends on `test-utils`. Happy to change to `implementation` if you prefer

Reply:
Sounds good.